### PR TITLE
updates for chia-blockchain-1.8.1

### PR DESCRIPTION
**Pull request by geoffwalmsley**
- Update setup.py to include chia-blockchain 1.8.1
- Use changed output from `conditions_dict_for_solution`

### CODE REVIEW

File: src/cli/main.py [hunk 1]
```diff
@@ -131,9 +132,10 @@ def create_cmd(
     \b
     Make a transaction to create a clawback coin
     """
-    amount = int(float(amount) * MOJO_CONST)
-    fee = int(float(fee) * MOJO_CONST)
-    async def do_command(fingerprint):
+    final_amount = int(Decimal(amount) * MOJO_CONST)
```
Comment (altendky) [inline src/cli/main.py:135]: I might go the other way here.  Rename `amount` to `amount_str` or `raw_amount`.  You can assign an arbitrary parameter name to an option by passing that string after the flag strings.  `"-m", "--fee", "raw_fee",`.  Someday we'll just make Click types for these things so it's all embedded with the checks and conversions and so on.

File: src/drivers/cb_manager.py [hunk 2]
```diff
@@ -310,12 +307,10 @@ async def sign_coin_spends(self, coin_spends: List[CoinSpend]) -> SpendBundle:
...
             synthetic_secret_key = calculate_synthetic_secret_key(private_key, DEFAULT_HIDDEN_PUZZLE_HASH)
 
-            err, conditions_dict, cost = conditions_dict_for_solution(
+            conditions_dict = conditions_dict_for_solution(
                 coin_spend.puzzle_reveal, coin_spend.solution, DEFAULT_CONSTANTS.MAX_BLOCK_COST_CLVM
             )
-            if err or conditions_dict is None:
-                error_msg = f"Sign transaction failed, con:{conditions_dict}, error: {err}"
-                raise ValueError(error_msg)
+            assert conditions_dict
```
Comment (altendky) [inline src/drivers/cb_manager.py:313]: What is this needed for?

**Review by altendky — CHANGES_REQUESTED** [review]

File: src/cli/main.py (hunk 1 above)
Comment (geoffwalmsley) [reply src/cli/main.py]: Thanks, fixed

**Comment by altendky** [comment]
Looks like there are a couple other places the `fee_str` pattern could be applied.

**Review by altendky — APPROVED** [review]
Thanks!

**Review by paninaro — APPROVED** [review]

**Pull request merged**